aarch64/src/main.rs

```diff
@@ -118,19 +110,14 @@ pub extern "C" fn main9(dtb_va: usize) {
     println!("midr_el1: {:?}", registers::MidrEl1::read());
 
     print_binary_sections();
-    print_physical_memory_info();
     print_board_info();
 
     pagealloc::init_page_allocator();
 
     // Map address space accurately using rust VM code to manage page tables
     unsafe {
         let dtb_range = PhysRange::with_len(from_virt_to_physaddr(dtb_va).addr(), dt.size());
-        vm::init_kernel_page_tables(
-            &mut *ptr::addr_of_mut!(KERNEL_PAGETABLE),
-            dtb_range,
-            mailbox::get_arm_memory(),
-        );
+        vm::init_kernel_page_tables(&dt, &mut *ptr::addr_of_mut!(KERNEL_PAGETABLE), dtb_range);
```

Comment:
I think passing both `dt` and `dtb_range` is redundant.
How do you feel about a struct that holds both the range and the parsed DT?